static/src/stylesheets/module/commercial/glabs/_hosted-gallery.scss

```diff
@@ -62,10 +66,15 @@
     .hosted-gallery__image-sizer {
         position: absolute;
         overflow: hidden;
+        width: 100%;
+        height: 100%;
     }
     .u-responsive-ratio {
         position: static;
     }
+    img {
+        object-fit: cover;
```

Comment:
How does the gallery look and work in IE? The `object-fit: cover` is not supported there.